lib/log-processors.js

```diff
@@ -0,0 +1,96 @@
+const LogProcessor = require('./LogProcessor')
```

Comment:
Can you please make sure that there is ‘use strict’ at the top of each file?

Reply:
Sorry about that. Done.